### PR TITLE
fix(cmd/alias): fix select menus

### LINKED ISSUE #56: discord.js 13 breaking everything yay
discord.js 13 replaced the button system with _a way worse one_ and now we have to fix our stuff
- [x] &poll (buttons.js)
- [x] &roshambo (rock paper scissors.js)
- [ ] &alias (parsers.js)

there may be more, let me know if you find them

### PR DESCRIPTION
**Pull request by LJMSU**
This PR:
- Fixes select menus on &alias delete

Fixes #56

### CODE REVIEW

File: src/parsers.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const { RequiredArg, Command } = require("./commands.js")
 const fs = require('fs')
-const { Console } = require("console")
```
Comment (Felix-44) [inline src/parsers.js:3]: what is this line for?

**Review by Felix-44 — APPROVED** [review]
as usual your code is great but...

File: src/parsers.js (hunk 1 above)
Comment (LJMSU) [reply src/parsers.js:3]: idk it was unused so i deleted it

Comment (Felix-44) [reply src/parsers.js:3]: oh lol didn't even notice it was removed and not added, oops

**Pull request merged**